core-bundle/src/Resources/contao/modules/ModuleRegistration.php

```diff
@@ -419,7 +419,7 @@ protected function createNewUser($arrData)
 			if ($objHomeDir !== null)
 			{
 				$this->import(Files::class, 'Files');
-				$strUserDir = StringUtil::standardize($arrData['username']) ?: 'user_' . $objNewUser->id;
+				$strUserDir = StringUtil::standardize($arrData['username'] ?? '') ?: 'user_' . $objNewUser->id;
```

Comment:
```suggestion
				$strUserDir = !empty($arrData['username']) ? StringUtil::standardize($arrData['username']) : 'user_' . $objNewUser->id;
```

Just use the regular ternary operator instead of combining two shorthand operators. Makes the code much easier to understand.

Reply:
But your suggestion would change the behavior, as `StringUtil::standardize()` might return an empty string even if `$arrData['username']` is not empty.